src/Tests/Microsoft.Diagnostics.Monitoring.HostingStartup.UnitTests/ParameterCapturing/ObjectFormatting/Formatters/DebuggerDisplay/DebuggerDisplayFormatterTests.cs

```diff
@@ -75,10 +84,31 @@ public void GetDebuggerDisplayFormatter_ReturnsWorkingFormatter()
             Assert.NotNull(factoryResult);
 
             // Act
-            string formattedResult = factoryResult.Formatter(testObj);
+            ObjectFormatterResult formattedResult = factoryResult.Formatter(testObj);
+
+            // Assert
+            Assert.Equal(FormattableString.Invariant($"'Count = {testObj.Count}'"), formattedResult.FormattedValue);
+            Assert.Equal(ParameterEvaluationFlags.None, formattedResult.Flags);
+        }
+
+        [Fact]
+        public void GetDebuggerDisplayFormatter_FormatsNull()
```

Comment:
This test is really testing behavior of `ExpressionBinder` and not `GetDebuggerDisplayFormatter`. Let's update the `ExpressionBinderTests` instead with this new null case.

I'd recommend just adding a new `InlineData` for this test case (and updating `DebuggerDisplayClass` with a new method/property that is null.
https://github.com/dotnet/dotnet-monitor/blob/efbb575789e1137fb4416a33a3279ab64a26d38a/src/Tests/Microsoft.Diagnostics.Monitoring.HostingStartup.UnitTests/ParameterCapturing/ObjectFormatting/Formatters/DebuggerDisplay/ExpressionBinderTests.cs#L55

Reply:
Nevermind, looks like I misread what got changed here!